src/modules/collection/constants.ts

```diff
@@ -1 +1,2 @@
 export const MAX_ITEMS = 50
+export const MAX_TP_ITEMS_TO_SHOW = 50
```

Comment:
Nitpick: I think the name here is a bit misleading. We're using it to filter items to show on a particular component.
MAX_ITEMS is the max amount of items to publish (could/should be called MAX_PUBLISH_ITEMS) but allows for it to be used in different scenarios.

Two things come to mind:
- Rename both to MAX_PUBLISH_ITEMS_AMOUNT and MAX_PUBLISH_TP_ITEMS_AMOUNT or something of the sort
- Rename MAX_TP_ITEMS_TO_SHOW to MAX_TP_ITEMS

What to you think (I couldn't come up with a better name for the first option 😅 )

Reply:
agreed, I'm sticking with `MAX_TP_ITEMS`!